pom.xml

```diff
@@ -542,6 +542,7 @@
       <plugin>
         <groupId>org.apache.rat</groupId>
         <artifactId>apache-rat-plugin</artifactId>
+        <version>0.16.1</version>
```

Comment:
We should bump this in `logging-parent` if necessary or even better in the ASF parent: apache/maven-apache-parent#193